src/map.rs

```diff
@@ -3093,11 +3093,14 @@ where
     S: BuildHasher + Clone,
 {
     fn clone(&self) -> HashMap<K, V, S> {
-        let cloned_map = Self::with_capacity_and_hasher(self.len(), self.build_hasher.clone());
+        let cloned_map = Self::with_capacity_and_hasher(self.len(), self.build_hasher.clone())
+            .with_collector(self.collector.clone());
```

Comment:
Does this create a new collector, or a new handle to the same collector? If it's the latter, then this would preclude us from allowing `K` and `V` to be `!: 'static`.

Reply:
It creates a new collector with the same settings.

Reply:
Ah, okay, great!